src/PiWeb.Import.Sdk/Modules/ImportAutomation/IImportAutomation.cs

```diff
@@ -8,69 +8,71 @@
 
 #endregion
 
-namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportAutomation;
+#region usings
+
+using Zeiss.PiWeb.Import.Sdk.PropertyStorage;
 
-using System.Threading.Tasks;
-using ConfigurationItems;
-using PropertyStorage;
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportAutomation;
 
 /// <summary>
 /// Represents a custom import automation provided as part of a plugin. Custom import automations substitute the full Auto Importer
 /// import pipeline with custom logic and are available as import sources in an import plan configuration.
 /// </summary>
 public interface IImportAutomation
 {
-	#region methods
+    #region methods

```

Comment:
For some reason the indentation changed from tabs to spaces in that file.

Reply:
I think this is the auto formatting for this project. All other files are also indented with spaces. This file was not for some reason. We can change the general indentation but not in this PR.